gitdb/db/loose.py

```diff
@@ -228,9 +229,23 @@ def store(self, istream):
                 rename(tmp_path, obj_path)
             # end rename only if needed
 
-            # make sure its readable for all ! It started out as rw-- tmp file
-            # but needs to be rwrr
-            chmod(obj_path, self.new_objects_mode)
+            # Ensure rename is actually done and file is stable
+            # Retry up to 10 times - exponential wait & retry in ms.
+            for exp_backoff in [1, 4, 9, 16, 25, 36, 49, 64, 81, 100, 121, 144, 169, 196, 225, 256, 289]:
```

Comment:
Would you update the comment to say what matters here? Is it to run this up to 10 times (now there are 17 entries for the backoff), or to not wait for longer than X ms?

Could you also call `exp_backoff` -> `exp_backoff_ms`? Having no units on top of having no static typing is more than I can bear 😅.

Reply:
Done ;)